src/ks-app/src/app/containers/drag-and-drop/dnd.component.scss

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
```

Comment:
If no CSS is required for the demo, then we should remove the file itself.

Reply:
Will keep it now because I will probably use to show the demos in KS-app.